### PR TITLE
Feat/multilingual support

### PR DESCRIPTION
**Pull request by valfaro123**
### 🔥 Summary
For ticket [#114](https://app.zenhub.com/workspaces/engineering-l2-62618a6423afe100128e4656/issues/gh/casper-network/casper-blockexplorer/114)

### 😤 Problem / Goals
-We want to be add support for multiple languages 

### 🤓 Solution
-Use i18next to allow toggling of languages

### CODE REVIEW

File: frontend/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx [hunk 1]
```diff
@@ -31,33 +33,33 @@ export const BlockDetailsCard: React.FC<BlockDetailsCardProps> = ({
     deployHashes,
     rawBlock,
   } = block;
-
+  const { t } = useTranslation();
   return (
```
Comment (DorianKwan) [inline frontend/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx]: NIT: up to you

```suggestion
  const { t } = useTranslation();

  return (
```

File: frontend/src/components/layout/Home/BlocksInfo/BlocksInfo.tsx [hunk 2]
```diff
@@ -22,24 +23,27 @@ export const BlocksInfo: React.FC<BlockInfoProps> = ({
...
-      <H3Data>{blockHeight}</H3Data>
-      <DataContext>{blockEraTimeStamp}</DataContext>
-      <H3>Current Era</H3>
-      <H3Data>{blockEraID}</H3Data>
-    </BlockDetails>
-  </BlockInfoDisplay>
-);
+}) => {
+  const { t } = useTranslation();
+  return (
```
Comment (DorianKwan) [inline frontend/src/components/layout/Home/BlocksInfo/BlocksInfo.tsx]: NIT:
```suggestion
  const { t } = useTranslation();

  return (
```

File: frontend/src/components/layout/Home/DeployInfo/DeploysInfo.tsx [hunk 3]
```diff
@@ -12,24 +13,27 @@ import {
...
-      <H3Data>n/a</H3Data>
-      <DataContext>n/a</DataContext>
-      <H3>Today</H3>
-      <H3Data>n/a</H3Data>
-    </DeployDetails>
-  </DeploysInfoDisplay>
-);
+export const DeploysInfo: React.FC = () => {
+  const { t } = useTranslation();
+  return (
```
Comment (DorianKwan) [inline frontend/src/components/layout/Home/DeployInfo/DeploysInfo.tsx]: NIT:k
```suggestion
  const { t } = useTranslation();

  return (
```

File: frontend/src/components/layout/Home/PeerInfo/PeersInfo.tsx [hunk 4]
```diff
@@ -15,21 +16,24 @@ interface PeersInfoProps {
...
-    </PeersHeader>
-    <PeersDetails>
-      <H3>Currently online</H3>
-      <H3Data>{currentPeers}</H3Data>
-    </PeersDetails>
-  </PeersInfoDisplay>
-);
+export const PeersInfo: React.FC<PeersInfoProps> = ({ currentPeers }) => {
+  const { t } = useTranslation();
+  return (
```
Comment (DorianKwan) [inline frontend/src/components/layout/Home/PeerInfo/PeersInfo.tsx]: NIT:
```suggestion
  const { t } = useTranslation();

  return (
```

**Review by DorianKwan — APPROVED** [review]
Looks good, just a few instances where I think it might improve readability.  See nit comments.

Essentially just adding an extra line before jsx returns

Let me know if you resolve the test CI failures

**Review by hoffmannjan — APPROVED** [review]
LGTM

**Pull request merged**